<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.3.353">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Jess Martin, Reese Hotten-Somers, Nicole Merullo, Brooke Rothamer">

<title>LabNotebook_GutMicrobiome</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
</style>


<script src="LabNotebook_GutMicrobiome_files/libs/clipboard/clipboard.min.js"></script>
<script src="LabNotebook_GutMicrobiome_files/libs/quarto-html/quarto.js"></script>
<script src="LabNotebook_GutMicrobiome_files/libs/quarto-html/popper.min.js"></script>
<script src="LabNotebook_GutMicrobiome_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="LabNotebook_GutMicrobiome_files/libs/quarto-html/anchor.min.js"></script>
<link href="LabNotebook_GutMicrobiome_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="LabNotebook_GutMicrobiome_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="LabNotebook_GutMicrobiome_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="LabNotebook_GutMicrobiome_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="LabNotebook_GutMicrobiome_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">


</head>

<body class="fullcontent">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">

<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">LabNotebook_GutMicrobiome</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Author</div>
    <div class="quarto-title-meta-contents">
             <p>Jess Martin, Reese Hotten-Somers, Nicole Merullo, Brooke Rothamer </p>
          </div>
  </div>
    
  
    
  </div>
  

</header>

<section id="key" class="level2">
<h2 class="anchored" data-anchor-id="key">Key</h2>
<p>Initials of lab group member will be used to denote who wrote the entries <br> JM - Jess Martin <br> RHS - Reese Hotten-Somers <br> NM - Nicole Merullo <br> BR - Brooke Rothamer</p>
</section>
<section id="background" class="level2">
<h2 class="anchored" data-anchor-id="background">Background</h2>
</section>
<section id="project-specific-papers" class="level2">
<h2 class="anchored" data-anchor-id="project-specific-papers">Project Specific Papers</h2>
<p>Listed in alphabetical order, not in reading order.</p>
<p>Clayton et al.&nbsp;2018. The gut microbiome of nonhuman primates: Lessons in ecology and evolution. American Journal of Primatology 80(6): e22867.</p>
<p>Elie et al.&nbsp;2023. Comparison of DNA extraction methods for 16S rRNA gene sequencing in the analysis of the human gut microbiome. Scientific Reports 13: 10279.</p>
<p>Hayakawa et al.&nbsp;2018. Improving the standards for gut microbiome analysis of fecal samples: insights from the field biology of Japanese macaques on Yakushima Island. Primates 59: 423‚Äì436.</p>
<p>Song et al.&nbsp;2016. Preservation Methods Differ in Fecal Microbiome Stability, Affecting Suitability for Field Studies. mSystems 1(3): 10.1128/msystems.00021-16.</p>
<p>Videnska et al.&nbsp;2019. Stool sampling and DNA isolation kits affect DNA quality and bacterial composition following 16S rRNA gene sequencing using MiSeq Illumina platform. Scientific Reports 9: 13837.</p>
</section>
<section id="extraction-protocol" class="level2">
<h2 class="anchored" data-anchor-id="extraction-protocol">Extraction Protocol</h2>
<section id="important-notes" class="level3">
<h3 class="anchored" data-anchor-id="important-notes">Important Notes</h3>
<ul>
<li>When conducting this protocol, always wear a suitable lab coat, disposable acrylamide gloves and protective goggles/glasses. For more information, consult the appropriate safety data sheets (SDSs) available from the product supplier. <br></li>
<li>To ensure proper sample identification, label sides and tops of the tubes. Processes used in this protocol may rub the labels off of the tubes, so it‚Äôs important to duplicate! <br></li>
<li>Always include a control sample of sterile/DNAse free water for every extraction so that you can detect potential contamination! <br></li>
<li>Perform all centrifugation steps at room temperature (15‚Äì25¬∞C). <br></li>
<li>If Solution C1 has precipitated, heat at 60¬∞C until the precipitate dissolves. Shake to mix Solution C4 before use. <br></li>
</ul>
</section>
<section id="protocol" class="level3">
<h3 class="anchored" data-anchor-id="protocol">Protocol</h3>
<p>Last updated 18FEB24 by CAS</p>
<section id="required-equipment-and-reagents" class="level4">
<h4 class="anchored" data-anchor-id="required-equipment-and-reagents">Required Equipment and Reagents</h4>
<ul>
<li>In this protocol, equipment available in the <em>DNeasy PowerLyzer PowerSoil Kit</em> will be marked in <em>italics</em>. Equipment available in the <strong>SMAGL</strong> will be in <strong>bold</strong>. <br></li>
<li>Biosafety cabinet (BSC) - remember to prepare it before getting started! <strong>[BSC]</strong>. Unless otherwise specified, all steps of this protocol should be performed in the BSC. <br></li>
<li>Microcentrifuge set to 10,000 rcf (not rpm) <strong>[Centrifuge Workstation]</strong> <br></li>
<li>Pipettes and pipette tips (sterile filter tips), prepare ahead to make sure you don‚Äôt run out! [Use filter tips; available on shelving above <strong>PCR</strong> and <strong>Quantitation Workstations</strong>; you should ONLY use the pipettes in the <strong>BSC</strong> and/or <strong>Extraction Workstation</strong> for extractions. <strong>The Quantitation Station</strong> has its own set of pipettes.] <br></li>
<li>Heat block or water bath set at 65C (this takes time to heat up; get it ready before starting the protocol) <strong>[Extraction Workstation]</strong> <br></li>
<li>Vortexer <strong>[Extraction Workstation or BSC]</strong> <br></li>
<li>Bead-beater (we use an alternative to the PowerLyzer homogenizer) <strong>[BSC]</strong> <br></li>
</ul>
<p>Prior to starting, UV all tubes, pipettes, tips, etc. in the PCR or BSC for 10 minutes <br></p>
</section>
<section id="protocol-step-by-step" class="level4">
<h4 class="anchored" data-anchor-id="protocol-step-by-step">Protocol step by step</h4>
<ol type="1">
<li>Add 0.25 g of fecal matter to the <em>PowerBead Tubes</em>. <br></li>
<li>Add 750 ùúál of <em>PowerBead Solution</em> to the <em>PowerBead Tubes</em>. <br></li>
<li>Add 60 ùúál of <em>Solution C1</em> to tubes and <strong>vortex</strong> briefly. If Solution C1 has precipitated, heat to 60¬∫C until precipitate has dissolved (in heat block or water bath). <br></li>
</ol>
<ul>
<li>Note: Vortexing mixes the components in the <em>PowerBead Tube</em> and begins to disperse the sample.</li>
</ul>
<ol start="4" type="1">
<li>Incubate <em>PowerBead Tubes</em> at 65¬∫C for 30 min in <strong>heat block</strong> or <strong>water bath</strong>. <br></li>
</ol>
<ul>
<li>This may be done by adding the <strong>Extraction Workstation</strong> dry block to the <strong>BSC</strong>; if you choose to do this, please clean it and add it to the BSC while preparing the BSC prior to beginning the protocol]. <br></li>
</ul>
<ol start="5" type="1">
<li>Secure tubes in the <strong>Bead Beater</strong> <strong>[BSC]</strong> and run for 5 minutes at 2000 RPM, pause for 30 seconds, and run again for 5 minutes at 2000 RPM. You may run these in evenly divided batches to initiate the subsequent centrifuge step with balanced samples. <br></li>
<li><strong>Centrifuge</strong> tubes at 10,000 g (rcf) for 30 s. Make sure the <em>PowerBead Tubes</em> rotate freely in the centrifuge without rubbing. <br></li>
</ol>
<ul>
<li>If 30 s is not long enough to pellet the sample, you may run for 3 min at the same speed. <br></li>
</ul>
<ol start="7" type="1">
<li>Transfer supernatant (~400-500 ùúál) to clean <em>2 ml Collection Tube</em> (you may still have chunks of poo at this point in the supernatant). <br></li>
<li>Add 250 ùúál of <em>Solution C2</em> to tubes and <strong>vortex</strong> for 5 s. Incubate at 4¬∫C for 10 min. <br></li>
</ol>
<ul>
<li>Note: <em>Solution C2</em> has Inhibitor Removal Technology (IRT). It contains a reagent that can precipitate non-DNA organic and inorganic material, including humic substances, cell debris and proteins. It is important to remove contaminating organic and inorganic matter that may reduce DNA purity and inhibit downstream DNA applications. <br></li>
</ul>
<ol start="9" type="1">
<li><strong>Centrifuge</strong> the tubes at 10,000 g (rcf) for 1 min. <br></li>
<li>Transfer up to 600 ùúál of supernatant to a clean <em>2 ml Collection Tube</em> (make sure to only transfer the liquid ‚Äì none of the pellet here!). <br></li>
</ol>
<ul>
<li>Note: The pellet at this point contains non-DNA organic and inorganic material, including humic acid, cell debris and proteins. For best DNA yields and quality, avoid transferring any of the pellet. <br></li>
</ul>
<ol start="11" type="1">
<li>Add 200 ùúál of <em>Solution C3</em> to the tubes and <strong>vortex</strong> briefly. Incubate at 4¬∫C for 10 min (use the old mini-fridge for this step, below the PCR Workstation). <br></li>
</ol>
<ul>
<li>Note: <em>Solution C3</em> has Inhibitor Removal Technology (IRT) and is a second reagent to precipitate additional non-DNA organic and inorganic material including humic acid, cell debris and proteins. It is important to remove contaminating organic and inorganic matter that may reduce DNA purity and inhibit downstream DNA applications. <br></li>
</ul>
<ol start="12" type="1">
<li><strong>Centrifuge</strong> the tubes at 10,000 g (rcf) for 1 min. <br></li>
<li>Transfer up to 750 ùúál of supernatant to a clean <em>2 ml Collection Tube</em>. Once again, be careful to avoid the pellet! <br></li>
</ol>
<ul>
<li>Note: The pellet at this point contains non-DNA organic and inorganic material including humic acid, cell debris and proteins. For best DNA yields and quality, avoid transferring any of the pellet. <br><br>
</li>
</ul>
<ol start="14" type="1">
<li>Add 1.2 ml of <em>Solution C4</em> to the supernatant and <strong>vortex</strong> for 5 s. Shake <em>Solution C4</em> by hand to mix prior to adding. Also, this will fill the <em>Collection Tube</em> to the brim, so be prepared. <br></li>
</ol>
<ul>
<li>Note: This contains guanidine salts, which is highly reactive. Do not mix or clean with bleach!!! <br></li>
<li>Note: <em>Solution C4</em> is a high-concentration salt solution. Since DNA binds tightly to silica at high salt concentrations, this will adjust the DNA solution salt concentrations to allow binding of DNA, but not non-DNA organic and inorganic material that may still be present at low levels, to the <em>Spin Filter</em>. <br></li>
</ul>
<ol start="15" type="1">
<li>Three washes of supernatant in <em>Spin Filter</em> <br></li>
</ol>
<ol type="a">
<li>First wash: Pipette 675 ùúál of the supernatant to a <em>Spin Filter</em> and <strong>centrifuge</strong> at 10,000 g (rcf) for 1 min. Discard the liquid flow through. Repeat until all supernatant has passed through the column. (This will take 3x washes through the column). <br></li>
</ol>
<ul>
<li>Note: DNA is selectively bound to the silica membrane in the <em>MB Spin Column</em> device in the high salt solution. Contaminants pass through the filter membrane, leaving only DNA bound to the membrane. <br></li>
</ul>
<ol start="2" type="a">
<li>Second Wash: Pipette 675 ùúál of the supernatant to the same <em>Spin Filter</em> and <strong>centrifuge</strong> at 10,000 g (rcf) for 1 min. Discard the liquid flow through. <br></li>
<li>Third Wash: Pipette 675 ùúál of the supernatant to the same <em>Spin Filter</em> and <strong>centrifuge</strong> at 10,000 g (rcf) for 1 min. Discard the liquid flow through. <br></li>
</ol>
<ol start="16" type="1">
<li>Add 500 ùúál of <em>Solution C5</em> to the filter and <strong>centrifuge</strong> at 10,000 g (rcf) for 30 s. Discard the liquid flow through. <br></li>
</ol>
<ul>
<li>Note: <em>Solution C5</em> is an ethanol-based wash solution used to further clean the DNA that is bound to the silica filter membrane in the <em>Spin Filter</em>. This wash solution removes residual salt, humic acid, and other contaminants while allowing the DNA to stay bound to the silica membrane. <br></li>
</ul>
<ol start="17" type="1">
<li><strong>Centrifuge</strong> the <em>Spin Filter</em> column a second time at 10,000 g (rcf) for 1 min. <br></li>
</ol>
<ul>
<li>Note: This flow-through fraction is non-DNA organic and inorganic waste removed from the silica <em>Spin Filter</em> membrane by the ethanol wash solution. The second spin removes residual <em>Solution C5</em> (ethanol wash solution). It is critical to remove all traces of wash solution because the ethanol in <em>Solution C5</em> can interfere with many downstream DNA applications, such as PCR and gel electrophoresis. <br></li>
</ul>
<ol start="18" type="1">
<li>Place <em>Spin Filter</em> into a clean and labeled <em>2 ml Collection Tube</em> (this is the tube going into the freezer, so make sure it‚Äôs labeled appropriately). Avoid splashing any <em>Solution C5</em> onto the <em>Spin Filter</em>.<br></li>
<li>Incubate <em>Solution C6</em> for 5 minutes at 65¬∫C. Then, add 100 ùúál of <em>Solution C6</em> to the center of the filter membrane. <br></li>
</ol>
<ul>
<li>Note: Placing <em>Solution C6</em> in the center of the small white membrane will make sure the entire membrane is wet. This will result in a more efficient and complete release of the DNA from the Spin Filter membrane. As <em>Solution C6</em> passes through the silica membrane, DNA that was bound in the presence of high salt is selectively released by <em>Solution C6</em> (which is actually 10 mM Tris), which lacks salt. <br></li>
</ul>
<ol start="20" type="1">
<li>Incubate samples at room temperature for 25 minutes. Then <strong>centrifuge</strong> at 10,000 g (rcf) for 30 sec, and finally discard the spin filter. <br> Your DNA is ready for use! <br></li>
<li>Measure the dsDNA concentration using the <strong>Quantus</strong>, and sample A260/A280 using the <strong>NanoDrop</strong> <strong>[Quantitation Workstation]</strong>. Record these details in your lab notebook. <br> Place the properly labeled DNA elutions in the <strong>-20¬∫C freezer</strong>. Be sure to record where in the freezer you‚Äôve placed them, and that they are all together in a single, clearly labeled container/rack.</li>
</ol>
</section>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "Óßã";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    text: function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->



</body></html>