---
title: "LabNotebook_GutMicrobiome"
author: "Jess Martin, Reese Hotten-Somers, Nicole Merullo, Brooke Rothamer"
format: html
editor: visual
---

## Key

Initials of lab group member will be used to denote who wrote the entries <br> JM - Jess Martin <br> RHS - Reese Hotten-Somers <br> NM - Nicole Merullo <br> BR - Brooke Rothamer

## Schedules of Group Members

```{r}

# Load library
suppressWarnings(library(knitr))

# Create Nicole's schedule data frame
Nicole_schedule <- data.frame(
  Day = c("Monday", "Monday", "Monday", "Monday", 
          "Tuesday", "Wednesday", "Wednesday", "Wednesday", 
          "Thursday", "Friday", "Friday", "Friday"),
  Time = c("9:00a-10:00a", "10:00a-12:00p", "1:25p-2:15p", "2:30p-5:15p", 
           "10:00a-10:30a", "10:00a-12:00p", "12:20p-1:10p", "1:25p-2:15p", 
           "2:30p-3:30p", "10:00a-11:00a", "1:25p-2:15p", "2:30p-3:30p"),
  Event = c("Carolyn/Luke Lab Meeting", "AN902 Class", "AN730 Class", "AN704 Class", 
            "Meeting w/ Chris", "AN701 Class", "Anthro Dept Event of the Week", "AN730 Class", 
            "Meeting w/ Carolyn (Tentative)", "AN730 Seminar", "AN730 Class", "Meeting w/ RA")
)

# Print Nicole's table
kable(Nicole_schedule, align = c('l', 'l', 'l'), caption = "<b style='font-size: 20px;'>Nicole's Weekly Schedule</b>")

# Create Jess's schedule data frame
Jess_schedule <- data.frame(
  Day = c("Monday", "Monday", "Monday", 
          "Tuesday", "Tuesday", 
          "Wednesday", "Wednesday", 
          "Thursday", "Thursday", "Thursday", 
          "Friday", "Friday", "Friday", "Friday", "Friday"),
  Time = c("10:00a-12:00p", "2:30p-3:00p", "4:00p-5:00p", 
           "2:00p-3:15p", "3:00p-4:45p", 
           "8:00a-10:45a", "12:20p-1:10p", 
           "10:00a-12:00p", "2:00p-3:15p", "3:30p-6:00p", 
           "9:05a-9:55a", "12:00p-1:00p", "1:25p-2:15p", "2:30p-3:30p", "3:30p-5:00p"),
  Event = c("AN902 Class", "Meeting w/ Chris", "SMAGL Lab Meeting", 
            "AN263 TF Lecture", "BI611 Class", 
            "AN510 Class", "Anthro Dept Event of the Week", 
            "TF Office Hours", "AN263 TF Lecture", "BI611 Class", 
            "TF Discussion Section", "Cheryl’s Lab Meeting", "TF Discussion Section", "AN902 Class (Cheryl)", "TF Meeting")
)

# Print Jess's Table
kable(Jess_schedule, align = c('l', 'l', 'l'), caption = "<b style='font-size: 20px;'>Jess's Weekly Schedule</b>")

#Create Reese's schedule data frame
Reese_schedule <- data.frame(
  Day = c("Monday", "Monday", "Monday", "Monday",
          "Tuesday", "Tuesday", "Tuesday", "Tuesday", "Tuesday",
          "Wednesday", "Wednesday", 
          "Thursday", "Thursday",  
          "Friday", "Friday"),
  Time = c("10:00a-12:00p", "2:30p-3:20p", "4:00p-5:00p", "5:15p-6:00p",
           "9:30a-10:45a", "11:00a-12:15p", "1:30p-2:00p", "3:35p-4:25p", "5:00p-7:00p",
           "12:00p-1:00a", "2:30p-3:20p", 
           "9:30a-10:45a", "11:00a-12:15p",  
           "11:15a-2:00p", "2:30p-3:20p"),
  Event = c("AN902 Class", "LS211 Class", "SMAGL Lab Meeting", "WM Eboard Meeting",
            "AR210 Class", "WS233 Class", "Meeting w/ Chris", "WS233 Discussion", "Rehearsal",
            "Meeting", "LS211 Class", 
            "AR210 Class", "WS233 Class",  
            "AN555 Class", "LS211 Class")
)

# Print Reese's Table
kable(Reese_schedule, align = c('l', 'l', 'l'), caption = "<b style='font-size: 20px;'>Reese's Weekly Schedule</b>")

# Create Brooke's schedule data frame
Brooke_schedule <- data.frame(
  Day = c("Monday", "Monday", "Monday", 
          "Tuesday", "Tuesday", 
          "Wednesday", "Wednesday", 
          "Thursday", "Thursday", 
          "Friday", "Friday", "Friday"),
  Time = c("10:00a-12:00p", "12:00p-4:00p", "4:00p-5:00p", 
           "1:00p-3:00p", "3:35p-4:25p", 
           "8:00a-10:45a", "12:20p-1:10p", 
           "10:00a-11:00a", "1:00p-3:00p", 
           "10:00a-1:30p", "2:30p-3:20p", "3:30p-5:00p"),
  Event = c("AN902 Class", "Meetings and Office Hours", "SMAGL Lab Meeting", 
            "Human Ev and Health Class", "TF Discussion Section", 
            "AN510 Class", "Anthro Dept Event of the Week", 
            "Meeting", "Human Ev and Health Class", 
            "BS 845 Class", "TF Discussion Section", "TF Meeting")
)

# Print Brooke's Table
kable(Brooke_schedule, align = c('l', 'l', 'l'), caption = "<b style='font-size: 20px;'>Brooke's Weekly Schedule</b>")
```

## Background

## Project Specific Papers

Note: Listed in alphabetical order, not in reading order.

### **Noninvasive genetic sampling of endangered muriqui (Primates, Atelidae): efficiency of fecal DNA extraction.**

**Chaves, P. B., Paes, M. F., Mendes, S. L., Strier, K. B., Louro, I. D., & Fagundes, V. (2006). Noninvasive genetic sampling of endangered muriqui (Primates, Atelidae): efficiency of fecal DNA extraction. *Genetics and Molecular Biology*, 29(4), 750--754.**

**Background:** The genus *Brachyteles* represents the largest neotropical nonhuman primate with two endemic species found in the Brazilian Atlantic Rainforest: *B. hypoxanthus* (northern muriqui) and *B. arachnoides* (southern muriqui). Due to habitat fragmentation and deforestation, muriquis have been classified as endangered since 1982 and critically endangered since 2000. Molecular genetic analyses have been limited by sample availability, but recent advancements allow for DNA extraction from fecal samples.

This study aimed to test the reliability of fecal DNA for PCR applications in muriquis. Fecal samples from 28 individually identified muriquis were collected and processed for DNA extraction. Various factors affecting PCR efficiency, such as DNA quality, quantity, and the influence of bovine serum albumin (BSA), were investigated. PCR amplification targeting a mitochondrial DNA segment was successful in most cases, indicating the potential of fecal DNA for genetic studies. Results showed that DNA quality and quantity significantly influenced PCR efficiency, with higher success rates observed in less degraded and higher quantity DNA samples. However, even degraded samples could yield positive results under optimal conditions. Interestingly, the addition of BSA did not significantly affect amplification efficiency in this study. This study also demonstrated the efficacy of using a cheaper DNA polymerase alternative and optimizing primer concentrations to minimize nonspecific PCR bands. The resulting PCR products were suitable for further analysis using Restriction Fragment Length Polymorphisms.

**Conclusion:** This study provides valuable insights into optimizing PCR reactions using fecal DNA from endangered muriquis, potentially reducing costs and time in genetic studies. The protocol developed may also apply to a wide range of primate and non-primate mammals in future genetic research projects.

### **Associations Between Nutrition, Gut Microbiome, and Health in A Novel Nonhuman Primate Model**

**Clayton, J.B., Al-Ghalith, G.A., Long, H.T., Tuan, B.V., Cabana, F., Huang, H., Vangay, P., Ward, T., Minh, V.V., Tam, N.A., Dat, N.T., Travis, D.A., Murtaugh, M.P., Covert, H., Glander, K.E., Nadler, T., Toddes, B., Sha, J.C.M., Singer, R., Knights, D., Johnson, T.J. (2018). Associations Between Nutrition, Gut Microbiome, and Health in A Novel Nonhuman Primate Model. *Sci Rep*, 8(1), 11159.**

This study examines the fecal microbiomes of different populations of red-shanked doucs (*Pygathrix nemaeus*), a colobine primate species, living under various lifestyle conditions (wild, semi-wild, semi-captive, captive). This study aims to understand how lifestyle factors (diet and environment), influence the gut microbial communities of these primates, and how these microbial changes may relate to their health.

**Background:** The GI tract of primates, including humans, hosts trillions of bacteria that play crucial roles in various aspects of host health and behavior. While there has been considerable research on the human microbiome and microbiome studies in non-primate animals, less attention has been given to nonhuman primates, particularly concerning their lifestyle, diet, and gut microbiomes. The red-shanked douc is of particular interest due to its digestive characteristics, which combine foregut fermentation and hindgut digestion, resembling both humans and ruminant livestock. Additionally, doucs are endangered, and their failure to thrive in captivity, primarily due to severe gastrointestinal disease, presents a challenge.

**Methods:** Fecal samples were collected from doucs across all four populations. Samples were analyzed using DNA extraction, 16S rRNA PCR amplification, and Illumina MiSeq sequencing to characterize the fecal microbiomes.

**Results:**

·       Microbiome Diversity Decline: This study found a decline in fecal microbiome diversity from wild to captive environments, with the highest diversity observed in wild doucs and the lowest in captive doucs. Semi-wild and semi-captive doucs showed intermediate levels of diversity.

·       Alpha Diversity: Alpha diversity measures (e.g., Shannon diversity index), revealed that wild doucs had the most even microbiomes compared to semi-wild, semi-captive, and captive doucs.

·       Beta Diversity: Beta diversity analysis showed significant differences in gut microbiome composition between douc populations, indicating that each population had a unique microbiome influenced by their lifestyle.

·       Taxonomic Abundance Analysis: The fecal microbiomes of all douc populations were dominated by the phylum *Firmicutes*. However, there were differences in the abundance of certain taxa, with *Bacteroidetes* being more abundant in semi-captive and captive populations. Taxa such as *Bacteroides* and *Prevotella* showed a positive correlation with captivity level, while others like *Adlercreutzia* and *Anaerostipes* decreased with increasing captivity.

·       Firmicutes to Bacteroidetes Ratio: The ratio of *Firmicutes* to *Bacteroidetes* was highest in wild doucs and decreased with increasing captivity, suggesting a relationship between lifestyle and the microbial community's energy harvest capacity.

·       Functional Analysis: Functional analysis using PICRUSt revealed shifts in microbial functions in captive doucs, with increased protein metabolism and decreased fatty acid metabolism compared to wild doucs. Certain pathways related to antibiotics and xenobiotic degradation were also associated with captivity.

**Conclusion:** This study provides valuable insights into the complex interplay between lifestyle, gut microbial composition, and health in doucs, with potential implications for understanding similar relationships in humans and other animal species.

### **The gut microbiome of nonhuman primates: Lessons in ecology and evolution**

**Clayton, J.B., Gomez, A., Amato, K., Knights, D., Travis, D.A., Blekhman, R., Knight, R., Leigh, S., Stumpf, R., Wolf, T., Glander, K.E., Cabana, F., Johnson, T.J. (2018). The gut microbiome of nonhuman primates: Lessons in ecology and evolution. *Am J Primatol*, 80(6).**

**Introduction to Host-Microbiome Relationship:** The relationship between animals and their microbiomes is ancient, beginning when multicellular organisms evolved in a biosphere dominated by microbes. This relationship is particularly notable in the gastrointestinal tract, where microbial populations are abundant, diverse, and crucial for host health. The gut microbiome aids hosts in various functions, such as extracting energy from food and maintaining immune responses.

**Advancements in Microbiome Research:** Research on host-associated microbes traditionally relied on culture-dependent methods, which have limitations in identifying diverse microbial taxa. Culture-independent methods, such as DNA sequencing, have since revolutionized microbial ecology by providing a more comprehensive view of microbial communities. High-throughput sequencing platforms enable the analysis of large numbers of samples simultaneously, advancing our understanding of microbial diversity.

**Challenges in Microbiome Research:** Despite these advances, challenges persist in gut microbiome research, including biases introduced during sample collection, DNA extraction, and PCR amplification. Additionally, the choice of sequencing region and bioinformatics methods can influence results. Shotgun metagenomics offers an alternative approach that sequences all genetic material present in a sample, providing insights into the functional capabilities of gut microbial communities.

**Regional Variations in Microbiota Composition:** Feces are the most used material for studying the gut microbiome, but they may not fully represent microbial communities throughout the GI tract. Regional variations in microbiota composition have been observed, suggesting that fecal samples may not accurately reflect microbial populations in other gut regions.

**Importance of Studying Primate Gut Microbiome:** Understanding the primate gastrointestinal microbiome is essential for advancing knowledge of primate ecology, evolution, and health. Improved methodologies and interdisciplinary approaches will continue to enhance our understanding of host-microbiome interactions and their implications for animal biology and human health.

·       *Evolutionary Significance*: NHPs are crucial for understanding human evolution due to their genetic similarity to humans. The gut microbiota's symbiotic relationship with the host likely played a significant role in dietary adaptations, influencing primate specialization and gut physiology.

·       *Behavioral Implications*: There is evidence suggesting a link between gut microbiota and nervous system function, with potential implications for stress-related behavior and psychiatric disorders. However, understanding gut-brain communication in NHPs remains limited.

·       *Conservation Concerns*: Human population growth and activities such as deforestation pose significant threats to NHP habitats and populations, affecting their ranging patterns and increasing the risk of disease transmission. Rescue centers aim to rehabilitate and release NHPs affected by habitat loss, but captive diets may negatively impact gastrointestinal microbiomes, affecting health and welfare.

**Future Directions:** Large-scale studies with both cross-sectional and longitudinal samples are needed to further advance our understanding of primate gut microbiomes. Functional assessments of microbial communities and collaboration among researchers are crucial for establishing standard procedures and sharing comparable datasets across multiple species and ecological niches.

### **Comparison of DNA extraction methods for 16S rRNA gene sequencing in the analysis of the human gut microbiome**

**Elie, C., Perret, M., Hage, H., Sentausa, E., Hesketh, A., Louis, K., Fritah-Lafont, A., Leissner, P., Vachon, C., Rostaing, H., Reynier, F., Gervasi, G., & Saliou, A. (2023). Comparison of DNA extraction methods for 16S rRNA gene sequencing in the analysis of the human gut microbiome. *Scientific Reports*, 13(1), 10279.**

The reliability of metagenomics methods crucial for studying these microbial communities is often compromised by errors at various workflow stages, including sample collection, DNA extraction, library preparation, sequencing, and data analysis. To overcome these challenges and facilitate the integration of metagenomics methods into clinical practice, standardized procedures are urgently needed. Although some standardized protocols exist, the field of nucleic acid preparation is rapidly evolving, requiring regular protocol benchmarking. The choice of DNA extraction method significantly impacts bacterial community detection, making it a pivotal step in metagenomic studies.

This study evaluated four commercially available DNA extraction methods using 16S rRNA amplicon sequencing. These protocols underwent assessment based on both wet-lab and dry-lab criteria, including DNA yield, fragment size, purity, observed microbial diversity, accuracy in extracting Gram-positive bacteria, protocol repeatability, and overall performance. This study encompassed fecal samples from healthy volunteers, patients with Clostridium difficile infection (CDI), and a defined mixture of bacterial species (mock community) for accuracy assessment.

Results revealed significant variability in DNA extraction yield, fragment size, and purity among the tested protocols. Notably, the use of an upstream stool preprocessing device (SPD) improved DNA yield for some protocols and enhanced DNA purity for most. Additionally, combining SPD with DNA extraction protocols increased microbial diversity, particularly in recovering Gram-positive bacteria, as indicated by the *Firmicutes*/*Bacteroidetes* ratio. The accuracy of extraction protocols was further validated using a mock community, with improved predictions observed when SPD was employed. Protocol repeatability was also enhanced with SPD integration.

Overall, this study identified the DNeasy PowerLyzer PowerSoil kit (S-DQ) as the top-performing protocol, demonstrating superior performance across multiple criteria. While other protocols showed similarly good results for specific criteria, S-DQ consistently stood out as the most effective. These findings underscore the critical role of protocol selection in metagenomic studies and highlight the potential benefits of incorporating standardized preprocessing methods, such as SPD, to enhance DNA extraction efficiency and microbial community analysis.

### **Improving the standards for gut microbiome analysis of fecal samples: insights from the field biology of Japanese macaques on Yakushima Island**

**Hayakawa, T., Sawada, A., Tanabe, A. S., Fukuda, S., Kishida, T., Kurihara, Y., Matsushima, K., Liu, J., Akomo-Okoue, E.-F., Gravena, W., Kashima, M., Suzuki, M., Kadowaki, K., Suzumura, T., Inoue, E., Sugiura, H., Hanya, G., & Agata, K. (2018). Improving the standards for gut microbiome analysis of fecal samples: insights from the field biology of Japanese macaques on Yakushima Island. *Primates*, 59(5), 423-436.**

This study explores optimizing the analysis of gut microbiomes in wild Japanese macaques (*Macaca fuscata*) and captive ones, recognizing the significant impact of gut microbiome adaptation on various aspects of host biology. Employing next-generation sequencing based on the bacterial 16S ribosomal RNA (rRNA) gene, the research aimed to address the challenges posed by traditional fecal microbiome analysis techniques, which may not be directly applicable to wild nonhuman primates due to sample degradation, bacterial growth, and lack of laboratory facilities in remote habitats.

To tackle these challenges, this study explored various methods of sample storage, including freezing, stabilization solutions like RNAlater, and lysis buffer, to preserve fecal DNA effectively. By collecting fecal samples from both captive and wild Japanese macaques, this study compared different storage methods' effectiveness in preserving fecal DNA. DNA extraction, purification, library preparation, and amplicon sequencing were conducted following established protocols, with operational taxonomic unit (OTU) picking and taxonomic identification characterizing the gut microbiome composition.

**Results:**

·       *Experiment 1*: Evaluated the effects of various sample storage and DNA extraction methods on captive Japanese macaque fecal samples, revealing some differences in OTU numbers and alpha diversity between storage methods but overall consistency in bacterial composition.

·       *Experiment 2*: Extended these findings to wild Japanese macaque fecal samples, observing slight variations across different storage methods but consistent microbiome profiles overall.

·       *Experiment 3*: Focused on comparing microbiome analysis results obtained from sequencing different regions of the 16S rRNA gene (V1-V2 vs. V3-V4), showing that the choice of sequencing region influenced the number of OTUs obtained. However, the Shannon index, a measure of alpha diversity, remained similar between the two regions, underscoring the importance of primer selection in microbiome studies.

This study provides valuable insights into methodological considerations for studying gut microbiomes in wild primates, emphasizing the need for optimizing sample processing methods and careful primer selection for accurate microbiome analysis. These findings contribute to our understanding of host-microbiome interactions in natural habitats and pave the way for future research to explore the effects of environmental factors on microbiome composition in wild primate populations.

### **Preservation Methods Differ in Fecal Microbiome Stability, Affecting Suitability for Field Studies**

**Song, S. J., Amir, A., Metcalf, J. L., Amato, K. R., Xu, Z. Z., Humphrey, G., & Knight, R. (2016). Preservation Methods Differ in Fecal Microbiome Stability, Affecting Suitability for Field Studies. *mSystems*, 1(3).**

This study aimed to investigate the effects of different storage methods on fecal microbiota composition over 8 weeks using 10 human and 5 dog donors. Four common preservatives (ethanol at 70% and 95% concentrations, RNAlater, OMNIgene Gut, and FTA cards) were tested under various temperature conditions, and the microbial communities were analyzed using 16S rRNA amplicon sequencing.

**Small Storage Effects:** Despite variations introduced by storage methods, these effects were small compared to differences between species and individuals. The variation between species (human and dog) and among individuals explained a significant portion of the total variation in microbial communities.

**Preservation Methods and Community Shifts:** Certain preservation methods, such as freezing and the use of FTA cards, OMNIgene Gut, and 95% ethanol, resulted in minimal changes in microbial community composition and diversity, even after 8 weeks. Unstabilized samples and those preserved with 70% ethanol showed larger changes, especially under heat or freeze-thaw conditions.

**Stability of Preservation Methods:** FTA cards and OMNIgene Gut showed minimal changes in microbial community composition and diversity over time, similar to fresh samples. RNAlater and 95% ethanol also performed well, although RNAlater showed slightly more changes under non-freezing temperatures over longer periods.

**Taxonomic Biases Introduced by Preservation:** Different preservation methods introduced biases in specific taxa. For example, FTA cards resulted in increased diversity of Firmicutes taxa, while 70% ethanol led to increases in Streptococcus and Haemophilus taxa. OMNIgene Gut showed the smallest immediate changes in taxa abundance.

**Recommendations:** This study recommends using 95% ethanol, OMNIgene Gut, or FTA cards for long-term storage, cautioning against the use of 70% ethanol or unfixed samples for extended periods. The recommended methods showed minimal changes in microbial communities, comparable to technical replicates, making them suitable for microbiome analysis even over several weeks.

Overall, this study highlights the importance of selecting appropriate preservation methods for fecal samples in microbiome research, especially for long-term storage and field trials, to ensure minimal alterations in microbial community composition and accurate interpretation of results.

### **Stool sampling and DNA isolation kits affect DNA quality and bacterial composition following 16S rRNA gene sequencing using MiSeq Illumina platform**

**Videnska, P., Smerkova, K., Zwinsova, B., Popovici, V., Micenkova, L., Sedlar, K., & Budinska, E. (2019). Stool sampling and DNA isolation kits affect DNA quality and bacterial composition following 16S rRNA gene sequencing using MiSeq Illumina platform. *Sci Rep*, 9(1), 13837.**

Large cohort studies are essential for comprehensively understanding the microbiome's role, but they encounter obstacles related to sampling procedures and methodological variability. User-friendly sampling protocols are crucial for participant compliance, while methodological differences across studies can hinder cross-study comparisons.

To address these challenges, this study systematically assessed the effects of sampling and DNA isolation kits on bacterial DNA quality, diversity, and composition while considering user acceptance. Sixteen volunteers used three different sampling kits: stool container, flocked swabs, and cotton swabs. DNA was extracted using two isolation kits: PowerLyzer PowerSoil DNA Isolation Kit (PS) and QIAamp DNA Stool Mini Kit (QS).

The stool container emerged as the most acceptable sampling option, while the QS kit yielded significantly higher DNA yields compared to the PS kit. However, the PS kit produced DNA of better integrity. The addition of a bead-beating step to the QS protocol did not improve DNA integrity, suggesting other factors at play.

The presence of PCR inhibitors persisted in DNA samples after isolation with both kits, with stool container samples exhibiting lower proportions, indicating potential advantages for certain applications. Bacterial diversity analysis revealed that the PS kit yielded a higher number of observed OTUs, particularly when combined with the stool container sampling kit. The final bacterial composition was more influenced by the DNA isolation kit than the sampling kit, with the PS kit showing a preference for Gram-positive bacteria. This study underscores the importance of considering batch effects arising from the choice of isolation and sampling kits when comparing results between studies.

This study highlights the significant effects of DNA isolation and sampling kits on DNA quality and bacterial composition, emphasizing the need for standardized methodologies and careful consideration of batch effects in microbiome research.

### Host genetics influence the gut microbiome.

**Cortes-Ortiz & Amato. 2021. Host genetics influence the gut microbiome. Science 373(6551): 159-160.**

\[note: this publication is not the primary report of this study\]

This study found some heritability of gut microbiome traits in yellow baboons (admixed with anubis baboons) living in Amboseli, Kenya. Such traits include the presence or relative abundance of certain taxa and the overall composition of the gut microbiome as measured from fecal samples. The researchers analyzed an unprecedented sample size of 858 individuals over 14 years, and they posited that previous studies have underestimated microbiome heritability due to small sample sizes. Nonetheless, more variation in gut microbiome traits was attributable to environmental factors. Heritability estimates also varied with environmental factors including season, diet, and host age.

### Monitoring the variation in the gut microbiota of captive woolly monkeys related to changes in diet during a reintroduction process.

**Quiroga-Gonzalez et al. 2021. Monitoring the variation in the gut microbiota of captive woolly monkeys related to changes in diet during a reintroduction process. Scientific Reports 11: 6522.**

## Extraction Protocol

### Important Notes

-   When conducting this protocol, always wear a suitable lab coat, disposable acrylamide gloves and protective goggles/glasses. For more information, consult the appropriate safety data sheets (SDSs) available from the product supplier. <br>
-   To ensure proper sample identification, label sides and tops of the tubes. Processes used in this protocol may rub the labels off of the tubes, so it's important to duplicate! <br>
-   Always include a control sample of sterile/DNAse free water for every extraction so that you can detect potential contamination! <br>
-   Perform all centrifugation steps at room temperature (15--25°C). <br>
-   If Solution C1 has precipitated, heat at 60°C until the precipitate dissolves. Shake to mix Solution C4 before use. <br>

### Protocol

Last updated 18FEB24 by CAS

#### Required Equipment and Reagents

-   In this protocol, equipment available in the *DNeasy PowerLyzer PowerSoil Kit* will be marked in *italics*. Equipment available in the **SMAGL** will be in **bold**. <br>
-   Biosafety cabinet (BSC) - remember to prepare it before getting started! **\[BSC\]**. Unless otherwise specified, all steps of this protocol should be performed in the BSC. <br>
-   Microcentrifuge set to 10,000 rcf (not rpm) **\[Centrifuge Workstation\]** <br>
-   Pipettes and pipette tips (sterile filter tips), prepare ahead to make sure you don't run out! \[Use filter tips; available on shelving above **PCR** and **Quantitation Workstations**; you should ONLY use the pipettes in the **BSC** and/or **Extraction Workstation** for extractions. **The Quantitation Station** has its own set of pipettes.\] <br>
-   Heat block or water bath set at 65C (this takes time to heat up; get it ready before starting the protocol) **\[Extraction Workstation\]** <br>
-   Vortexer **\[Extraction Workstation or BSC\]** <br>
-   Bead-beater (we use an alternative to the PowerLyzer homogenizer) **\[BSC\]** <br>

Prior to starting, UV all tubes, pipettes, tips, etc. in the PCR or BSC for 10 minutes <br>

#### Protocol step by step

1.  Add 0.25 g of fecal matter to the *PowerBead Tubes*. <br>
2.  Add 750 𝜇l of *PowerBead Solution* to the *PowerBead Tubes*. <br>
3.  Add 60 𝜇l of *Solution C1* to tubes and **vortex** briefly. If Solution C1 has precipitated, heat to 60ºC until precipitate has dissolved (in heat block or water bath). <br>

-   Note: Vortexing mixes the components in the *PowerBead Tube* and begins to disperse the sample.

4.  Incubate *PowerBead Tubes* at 65ºC for 30 min in **heat block** or **water bath**. <br>

-   This may be done by adding the **Extraction Workstation** dry block to the **BSC**; if you choose to do this, please clean it and add it to the BSC while preparing the BSC prior to beginning the protocol\]. <br>

5.  Secure tubes in the **Bead Beater** **\[BSC\]** and run for 5 minutes at 2000 RPM, pause for 30 seconds, and run again for 5 minutes at 2000 RPM. You may run these in evenly divided batches to initiate the subsequent centrifuge step with balanced samples. <br>
6.  **Centrifuge** tubes at 10,000 g (rcf) for 30 s. Make sure the *PowerBead Tubes* rotate freely in the centrifuge without rubbing. <br>

-   If 30 s is not long enough to pellet the sample, you may run for 3 min at the same speed. <br>

7.  Transfer supernatant (\~400-500 𝜇l) to clean *2 ml Collection Tube* (you may still have chunks of poo at this point in the supernatant). <br>
8.  Add 250 𝜇l of *Solution C2* to tubes and **vortex** for 5 s. Incubate at 4ºC for 10 min. <br>

-   Note: *Solution C2* has Inhibitor Removal Technology (IRT). It contains a reagent that can precipitate non-DNA organic and inorganic material, including humic substances, cell debris and proteins. It is important to remove contaminating organic and inorganic matter that may reduce DNA purity and inhibit downstream DNA applications. <br>

9.  **Centrifuge** the tubes at 10,000 g (rcf) for 1 min. <br>
10. Transfer up to 600 𝜇l of supernatant to a clean *2 ml Collection Tube* (make sure to only transfer the liquid -- none of the pellet here!). <br>

-   Note: The pellet at this point contains non-DNA organic and inorganic material, including humic acid, cell debris and proteins. For best DNA yields and quality, avoid transferring any of the pellet. <br>

11. Add 200 𝜇l of *Solution C3* to the tubes and **vortex** briefly. Incubate at 4ºC for 10 min (use the old mini-fridge for this step, below the PCR Workstation). <br>

-   Note: *Solution C3* has Inhibitor Removal Technology (IRT) and is a second reagent to precipitate additional non-DNA organic and inorganic material including humic acid, cell debris and proteins. It is important to remove contaminating organic and inorganic matter that may reduce DNA purity and inhibit downstream DNA applications. <br>

12. **Centrifuge** the tubes at 10,000 g (rcf) for 1 min. <br>
13. Transfer up to 750 𝜇l of supernatant to a clean *2 ml Collection Tube*. Once again, be careful to avoid the pellet! <br>

-   Note: The pellet at this point contains non-DNA organic and inorganic material including humic acid, cell debris and proteins. For best DNA yields and quality, avoid transferring any of the pellet. <br>

14. Add 1.2 ml of *Solution C4* to the supernatant and **vortex** for 5 s. Shake *Solution C4* by hand to mix prior to adding. Also, this will fill the *Collection Tube* to the brim, so be prepared. <br>

-   Note: This contains guanidine salts, which is highly reactive. Do not mix or clean with bleach!!! <br>
-   Note: *Solution C4* is a high-concentration salt solution. Since DNA binds tightly to silica at high salt concentrations, this will adjust the DNA solution salt concentrations to allow binding of DNA, but not non-DNA organic and inorganic material that may still be present at low levels, to the *Spin Filter*. <br>

15. Three washes of supernatant in *Spin Filter* <br>

<!-- -->

a.  First wash: Pipette 675 𝜇l of the supernatant to a *Spin Filter* and **centrifuge** at 10,000 g (rcf) for 1 min. Discard the liquid flow through. Repeat until all supernatant has passed through the column. (This will take 3x washes through the column). <br>

-   Note: DNA is selectively bound to the silica membrane in the *MB Spin Column* device in the high salt solution. Contaminants pass through the filter membrane, leaving only DNA bound to the membrane. <br>

b.  Second Wash: Pipette 675 𝜇l of the supernatant to the same *Spin Filter* and **centrifuge** at 10,000 g (rcf) for 1 min. Discard the liquid flow through. <br>
c.  Third Wash: Pipette 675 𝜇l of the supernatant to the same *Spin Filter* and **centrifuge** at 10,000 g (rcf) for 1 min. Discard the liquid flow through. <br>

<!-- -->

16. Add 500 𝜇l of *Solution C5* to the filter and **centrifuge** at 10,000 g (rcf) for 30 s. Discard the liquid flow through. <br>

-   Note: *Solution C5* is an ethanol-based wash solution used to further clean the DNA that is bound to the silica filter membrane in the *Spin Filter*. This wash solution removes residual salt, humic acid, and other contaminants while allowing the DNA to stay bound to the silica membrane. <br>

17. **Centrifuge** the *Spin Filter* column a second time at 10,000 g (rcf) for 1 min. <br>

-   Note: This flow-through fraction is non-DNA organic and inorganic waste removed from the silica *Spin Filter* membrane by the ethanol wash solution. The second spin removes residual *Solution C5* (ethanol wash solution). It is critical to remove all traces of wash solution because the ethanol in *Solution C5* can interfere with many downstream DNA applications, such as PCR and gel electrophoresis. <br>

18. Place *Spin Filter* into a clean and labeled *2 ml Collection Tube* (this is the tube going into the freezer, so make sure it's labeled appropriately). Avoid splashing any *Solution C5* onto the *Spin Filter*.<br>

-   This is the final tube so it should be a low bind tube

19. Incubate *Solution C6* for 5 minutes at 65ºC. Then, add 100 𝜇l of *Solution C6* to the center of the filter membrane. <br>

-   Note: Placing *Solution C6* in the center of the small white membrane will make sure the entire membrane is wet. This will result in a more efficient and complete release of the DNA from the Spin Filter membrane. As *Solution C6* passes through the silica membrane, DNA that was bound in the presence of high salt is selectively released by *Solution C6* (which is actually 10 mM Tris), which lacks salt. <br>

20. Incubate samples at room temperature for 25 minutes. Then **centrifuge** at 10,000 g (rcf) for 30 sec, and finally discard the spin filter. <br> Your DNA is ready for use! <br>
21. Measure the dsDNA concentration using the **Quantus**, and sample A260/A280 using the **NanoDrop** **\[Quantitation Workstation\]**. Record these details in your lab notebook. <br> Place the properly labeled DNA elutions in the **-20ºC freezer**. Be sure to record where in the freezer you've placed them, and that they are all together in a single, clearly labeled container/rack.

## Extractions

NOTES: Please record by each step, above, who has done which step and when. Check with Prof. Schmitt prior to beginning regarding which samples should be extracted. ALWAYS make sure that there are sufficient reagents and equipment before beginning a protocol!

Sample ID Label: Final concentration (ng/uL) A260/A280 value

## Extraction Done 28Feb2024

-   Group members present: JM, BR, NM
-   Documented by NM, JM, BR

### Lab Preparation at 13:50

-   500 mL bleach spray (with DI water) prepared by BR
-   Heat block was turned on by JM
-   Centrifuge cleaned by JM
-   Bleach water alcohol the BSC by BR and JM
-   Made sure there were enough tubes (NM)
-   UV light tubes and tips in PCR hood started at 14:25
-   UV light pipettes in PCR hood started at 14:40 (JM)

### Sample Reconciliation at 15:40

-   Chris entered the original sample labels into the protocol table. Samples placed on ice.
-   JM and BR thawed the samples in hand and vortexed them. All samples were thawed as of 16:05. NM labeled the first collection tubes. JM double checked that the tubes were lined up in order.

### Reminders:

-   Do not reach over open tubes
-   Work from clean to dirty
-   Do not swing dirty pipettes over tubes
-   Hold sample tube cap in your hand and do not touch inside of cap
-   Close solution bottles in between samples
-   Do not put anything down without putting the cap back
-   Put caps on kimwipes if setting them down
-   Write sample name on side of the tube before bead beater step

### Protocol Step 1:

-   Add 0.25 g of fecal matter to the *PowerBead Tubes*

    -   16:18 JM and Chris added 250 ul of water to the control Powerbead Tube.
    -   16:20 JM added 250 ul of sample 01 to the 01 Powerbead Tube.
    -   16:31 Rearranged the BSC for the direction of dirty to clean right to left.
    -   16:33 For sample 02, it was coming up chunky. Discarded tip. Tip missed discard. Clean-up initiated.
    -   16:36 Cleaned a razor blade with bleach in the BSC and Chris trimmed a pipette tip.
    -   16:38 Chris added 250 ul of sample 02 to the 02 powerbead tube using the trimmed pipette tip.
    -   16:42 JM added \~250 ul sample 03 to the 03 powerbead tube.
    -   Voyt-18-017 (sample MB03) full volume was not drawn in, so possibly less starting volume than the other samples.
    -   16:44 Sample 04 was chunky. Discarded tip. Will now trim all tips.
    -   16:45 Cleaned razor blade with bleach and trimmed tip. JM added 250 ul of sample 04 to powerbead tube \*Samples were placed on ice after being aliquoted.
    -   16:47 Blade cleaned w/ bleach, trimmed tip, JM added 250 ul of sample 05 to the 05 powerbead tube. Sample 05 is the one that looks a little red (potentially blood).
    -   16:48 Blade cleaned w/ bleach, trimmed tip, JM added 250 ul of sample 06 to the 06 powerbead tube.
    -   16:49 Blade cleaned w/ bleach, trimmed tip, JM added 250 ul of sample 07 to the 07 powerbead tube.
    -   16:51 Blade cleaned w/ bleach, trimmed tip, JM added 250 ul of sample 08 to the 08 powerbead tube.
    -   16:52 Blade cleaned w/ bleach, trimmed tip, JM added 250 ul of sample 09 to the 09 powerbead tube.
    -   16:53 Blade cleaned w/bleach, trimmed tip, JM added 250 ul of sample 10 to the 10 powerbead tube.
    -   16:54 Blade cleaned w/ bleach, trimmed tip, JM added 250 ul of sample 11 to the 11 powerbead tube.

-   Original samples went back into the bottom shelf of the -20 freezer at 16:50. They were at room temperature from 16:05 to 16:50.

### Protocol Step 2:

-   Add 750 𝜇l of *PowerBead Solution* to the *PowerBead Tubes*.

    -   17:00 put Solution C1 on the heat block to warm up in anticipation of step 3
    -   17:00 JM put 750 ul of powerbead solution into sample CL
    -   17:01 JM put 750 ul of powerbead solution into sample 01
    -   17:02 JM put 750 ul of powerbead solution into sample 02
    -   17:02 JM put 750 ul of powerbead solution into sample 03
    -   17:03 JM put 750 ul of powerbead solution into sample 04
    -   17:03 JM put 750 ul of powerbead solution into sample 05
    -   17:04 JM put 750 ul of powerbead solution into sample 06
    -   17:05 JM put 750 ul of powerbead solution into sample 07
    -   17:05 JM glove ripped, NM stepped in to screw sample 08 cap back on while JM regloved
    -   17:06 JM put 750 ul of powerbead solution into sample 08
    -   17:06 JM put 750 ul of powerbead solution into sample 09
    -   17:07 JM put 750 ul of powerbead solution into sample 10
    -   17:07 JM put 750 ul of powerbead solution into sample 11

### Protocol Step 3

-   Add 60 𝜇l of *Solution C1* to tubes and **vortex** briefly. If Solution C1 has precipitated, heat to 60ºC until precipitate has dissolved (in heat block or water bath). <br>

-   Note: Vortexing mixes the components in the *PowerBead Tube* and begins to disperse the sample.

    -   17:11 BR put 60 ul of Solution C1 into sample CL and JM vortexed
    -   17:13 BR put 60 ul of Solution C1 into sample 01 and JM vortexed
    -   17:14 BR put 60 ul of Solution C1 into sample 02 and JM vortexed
    -   17:15 BR put 60 ul of Solution C1 into sample 03 and JM vortexed
    -   17:16 BR put 60 ul of Solution C1 into sample 04 and JM vortexed
    -   17:17 BR put 60 ul of Solution C1 into sample 05 and JM vortexed
    -   17:18 BR put 60 ul of Solution C1 into sample 06 and JM vortexed
    -   17:19 BR put 60 ul of Solution C1 into sample 07 and JM vortexed
    -   17:20 BR put 60 ul of Solution C1 into sample 08 and JM vortexed
    -   17:21 BR put 60 ul of Solution C1 into sample 09 and JM vortexed
    -   BR cleaned surface with bleach, water, alcohol after sample 09 cap made contact with the surface
    -   17:22 BR put 60 ul of Solution C1 into sample 10 and JM vortexed
    -   17:23 BR put 60 ul of Solution C1 into sample 11 and JM vortexed

### Protocol Step 4

-   Incubate *PowerBead Tubes* at 65ºC for 30 min in **heat block** or **water bath**.

-   This may be done by adding the **Extraction Workstation** dry block to the **BSC**; if you choose to do this, please clean it and add it to the BSC while preparing the BSC prior to beginning the protocol\].

    -   17:26 JM added tubes to the heat block and set a 30 minute timer
    -   17:56 JM pulled tubes from the heat block

### Protocol Step 5

-   Secure tubes in the **Bead Beater** **\[BSC\]** and run for 5 minutes at 2000 RPM, pause for 30 seconds, and run again for 5 minutes at 2000 RPM. You may run these in evenly divided batches to initiate the subsequent centrifuge step with balanced samples.

    -   17:58 All samples run on the Bead Beater for 5 minutes (JM)
    -   18:03 All samples rest for 30 seconds
    -   18:04 All samples run on the Bead Beater for 5 minutes (NM)

    \*18:09 NM noted that sample 08 looked odd- it had more foam and less liquid than the other samples.

### Protocol Step 6

-   **Centrifuge** tubes at 10,000 g (rcf) for 30 s. Make sure the *PowerBead Tubes* rotate freely in the centrifuge without rubbing.

-   If 30 s is not long enough to pellet the sample, you may run for 3 min at the same speed.

    -   18:12 All samples spun in the centrifuge for 30 seconds
    -   Check for pellet showed that 30 seconds was not long enough
    -   18:13 All samples spun in the centrifuge for 2 more minutes
    -   Check for pellet showed that 2 minutes was not long enough (The solids were at the bottom but beads still appeared to move freely)
    -   18:16 All samples spun in the centrifuge for 1 more minute

### Protocol Step 7

-   Transfer supernatant (\~400-500 𝜇l) to clean *2 ml Collection Tube* (you may still have chunks of poop at this point in the supernatant). <br>

    -   Using a pipette set to 500 ul

    -   18:26 NM transferred supernatant from CL (400 + 100 pipet)

    -   18:27 NM transferred supernatant from 01

    -   18:29 NM transferred supernatant from 02

    -   18:30 NM transferred supernatant from 03

    -   18:31 NM transferred supernatant from 04

    -   18:32 NM transferred supernatant from 05

    -   18:33 NM transferred supernatant from 06

    -   18:34 NM transferred supernatant from 07

    -   18:35 NM transferred supernatant from 08

    -   18:36 NM transferred supernatant from 09

    -   18:37 NM transferred supernatant from 10

    -   18:38 NM transferred supernatant from 11

    -   NM noted that sample 11 looked like the pellet was a little broken up before supernatant was taken up

### Protocol Step 8

-   Add 250 𝜇l of *Solution C2* to tubes and **vortex** for 5 s. Incubate at 4ºC for 10 min. <br>

-   Note: *Solution C2* has Inhibitor Removal Technology (IRT). It contains a reagent that can precipitate non-DNA organic and inorganic material, including humic substances, cell debris and proteins. It is important to remove contaminating organic and inorganic matter that may reduce DNA purity and inhibit downstream DNA applications. <br>

    -   18:42 JM added 250 ul solution C2 to the sample CL and BR vortexed for 5 sec

    -   18:43 JM added 250 ul solution C2 to the sample 1 and BR vortexed for 5 sec

    -   18:43 JM added 250 ul solution C2 to the sample 2 and BR vortexed for 5 sec

    -   18:44 JM added 250 ul solution C2 to the sample 3 and BR vortexed for 5 sec

    -   18:44 JM added 250 ul solution C2 to the sample 4 and BR vortexed for 5 sec

    -   18:44 JM added 250 ul solution C2 to sample 5 and BR vortexed for 5 sec \*JM noticed that the 1000 ul pipette was taking up a bubble each time and the white stem part was detaching from the base; NM had also noticed that it was getting increasingly more difficult to dispose of tips. JM switched to the smaller pipette which only goes up to 200 ul pipette. She is switching it between 200 and 50 ul for each sample.

    -   18:49 JM added 250 ul solution C2 to sample 6 and BR vortexed for 5 sec \*JM could not remember if she added 50 ul to sample 7, NM/JM/BR put it aside and used the pipette from the PCR station to measure if there is 750 ul in the tube. NM/JM/BR decided to use the 1000ul pipette from the PCR station, which had been sterilized earlier with the other materials. While this is not standard, NM/JM/BR agreed that these extenuating circumstances called for it, and making do with the smaller pipette was introducing unnecessary errors.

    -   18:52 JM added 250 ul solution C2 to sample 8 and BR vortexed for 5 sec

    -   18:52 JM added 250 ul solution C2 to sample 9 and BR vortexed for 5 sec

    -   18:53 JM added 250 ul solution C2 to sample 10 and BR vortexed for 5 sec

    -   18:53 JM added 250 ul solution C2 to sample 11 and BR vortexed for 5 sec \*JM measured sample 7 and found that there was less than 750 ul, so she will add the remaining 50 ul

    -   18:55 JM added 50 ul solution C2 to sample 7 and BR vortexed for 5 sec

        \*Put into 4 degrees celsius fridge at 18:56 and pulled at 19:56

### Protocol Step 9

-   **Centrifuge** the tubes at 10,000 g (rcf) for 1 min. <br>

    \*19:10 NM centrifuged the samples and checked pellets were formed. NM noted that sample 8 had a very small pellet compared to the rest.

### Protocol Step 10

-   Transfer up to 600 𝜇l of supernatant to a clean *2 ml Collection Tube* (make sure to only transfer the liquid -- none of the pellet here!). <br>

-   Note: The pellet at this point contains non-DNA organic and inorganic material, including humic acid, cell debris and proteins. For best DNA yields and quality, avoid transferring any of the pellet. <br>

    -   19:11 JM transferred 600 ul supernatant from sample CL to a clean 2 ml collection tube
    -   19:12 JM transferred 600 ul supernatant from sample 01 to a clean 2 ml collection tube
    -   19:12 JM transferred 600 ul supernatant from sample 02 to a clean 2 ml collection tube
    -   19:13 JM transferred 600 ul supernatant from sample 03 to a clean 2 ml collection tube
    -   19:13 JM transferred 600 ul supernatant from sample 04 to a clean 2 ml collection tube
    -   19:14 JM transferred 600 ul supernatant from sample 05 to a clean 2 ml collection tube
    -   19:14 JM transferred 600 ul supernatant from sample 06 to a clean 2 ml collection tube
    -   19:14 JM transferred 600 ul supernatant from sample 07 to a clean 2 ml collection tube
    -   19:15 JM transferred 600 ul supernatant from sample 08 to a clean 2 ml collection tube
    -   19:15 JM transferred 600 ul supernatant from sample 09 to a clean 2 ml collection tube
    -   19:15 JM transferred 600 ul supernatant from sample 10 to a clean 2 ml collection tube
    -   19:16 JM transferred 600 ul supernatant from sample 11 to a clean 2 ml collection tube

### Protocol Step 11

-   Add 200 𝜇l of *Solution C3* to the tubes and **vortex** briefly. Incubate at 4ºC for 10 min (use the old mini-fridge for this step, below the PCR Workstation). <br>

-   Note: *Solution C3* has Inhibitor Removal Technology (IRT) and is a second reagent to precipitate additional non-DNA organic and inorganic material including humic acid, cell debris and proteins. It is important to remove contaminating organic and inorganic matter that may reduce DNA purity and inhibit downstream DNA applications. <br>

    -   19:21 BR dropped sample CL on the bench, possible leakage

    -   19:21 BR transferred 200 ul of Solution C3 to sample CL into a clean 2 ml collection tube

    -   19:21 BR transferred 200 ul of Solution C3 to sample 01 into a clean 2 ml collection tube

    -   19:22 BR transferred 200 ul of Solution C3 to sample 02 into a clean 2 ml collection tube

    -   19:23 BR transferred 200 ul of Solution C3 to sample 03 into a clean 2 ml collection tube

    -   19:23 BR transferred 200 ul of Solution C3 to sample 04 into a clean 2 ml collection tube

    -   19:24 BR transferred 200 ul of Solution C3 to sample 05 into a clean 2 ml collection tube

    -   19:24 BR transferred 200 ul of Solution C3 to sample 06 into a clean 2 ml collection tube

    -   19:24 BR transferred 200 ul of Solution C3 to sample 07 into a clean 2 ml collection tube

    -   19:25 BR transferred 200 ul of Solution C3 to sample 08into a clean 2 ml collection tube

    -   19:25 BR transferred 200 ul of Solution C3 to sample 09 into a clean 2 ml collection tube

    -   19:26 BR transferred 200 ul of Solution C3 to sample 10 into a clean 2 ml collection tube

    -   19:26 BR transferred 200 ul of Solution C3 to sample 11 into a clean 2 ml collection tube

        \*Started incubation at 4ºC 19:26 for 10 minutes. Pulled at 19:36.

### Protocol Step 12

-   **Centrifuge** the tubes at 10,000 g (rcf) for 1 min. <br>

    -   19:39 All samples centrifuged for 1 min (JM)

### Protocol Step 13

-   Transfer up to 750 𝜇l of supernatant to a clean *2 ml Collection Tube*. Once again, be careful to avoid the pellet!

-   Note: The pellet at this point contains non-DNA organic and inorganic material including humic acid, cell debris and proteins. For best DNA yields and quality, avoid transferring any of the pellet.

    NM notes that ALL samples have less than 750 supernatant in tube.

    -   Started new 2ml collection tube
    -   19:44 NM transferred 750 ul supernatant from sample CL to a clean 2 ml collection tube
    -   19:45 NM transferred 750 ul supernatant from sample 01 to a clean 2 ml collection tube
    -   19:46 NM transferred 750 ul supernatant from sample 02 to a clean 2 ml collection tube
    -   19:47 NM transferred 750 ul supernatant from sample 03 to a clean 2 ml collection tube
    -   19:48 NM transferred 750 ul supernatant from sample 04 to a clean 2 ml collection tube
    -   19:49 NM transferred 750 ul supernatant from sample 05 to a clean 2 ml collection tube
    -   19:51 NM transferred 750 ul supernatant from sample 06 to a clean 2 ml collection tube
    -   NM breaks top off of new collection tube- BR hands her a new tube- NM labels and uses new tube
    -   19:52 NM transferred 750 ul supernatant from sample 07 to a clean 2 ml collection tube
    -   19:53 NM transferred 750 ul supernatant from sample 08 to a clean 2 ml collection tube
    -   19:54 NM transferred 750 ul supernatant from sample 09 to a clean 2 ml collection tube
    -   19:54 NM transferred 750 ul supernatant from sample 10 to a clean 2 ml collection tube
    -   19:55 NM transferred 750 ul supernatant from sample 11 to a clean 2 ml collection tube
    -   NM tipped over sample 11, pellet intact (no movement).

### Protocol Step 14

-   Add 1.2 ml of *Solution C4* to the supernatant and **vortex** for 5 s. Shake *Solution C4* by hand to mix prior to adding. Also, this will fill the *Collection Tube* to the brim, so be prepared. <br>

-   Note: This contains guanidine salts, which is highly reactive. Do not mix or clean with bleach!!! <br>

-   Note: *Solution C4* is a high-concentration salt solution. Since DNA binds tightly to silica at high salt concentrations, this will adjust the DNA solution salt concentrations to allow binding of DNA, but not non-DNA organic and inorganic material that may still be present at low levels, to the *Spin Filter*. <br>

-   20:00 NM added 1200 ul of solution C4 to sample CL. JM vortexed sample.

-   20:01 NM added 1200 ul of solution C4 to sample 01. JM vortexed sample.

-   20:03 NM added 1200 ul of solution C4 to sample 02. BR vortexed sample.

-   20:05 NM added 1200 ul of solution C4 to sample 03. BR vortexed sample.

-   20:06 NM added 1200 ul of solution C4 to sample 04. BR vortexed sample.

-   20:07 NM added 1200 ul of solution C4 to sample 05. BR vortexed sample.

-   20:07 NM added 1200 ul of solution C4 to sample 06. BR vortexed sample.

-   20:08 NM added 1200 ul of solution C4 to sample 07. BR vortexed sample.

-   20:09 NM added 1200 ul of solution C4 to sample 08. BR vortexed sample.

-   20:10 NM added 1200 ul of solution C4 to sample 09. BR vortexed sample.

-   20:11 NM added 1200 ul of solution C4 to sample 10. BR vortexed sample.

-   20:12 NM added 1200 ul of solution C4 to sample 11. BR vortexed sample.

### Protocol Step 15

Three washes of supernatant in *Spin Filter* <br>

<!-- -->

a\. First wash: Pipette 675 𝜇l of the supernatant to a *Spin Filter* and **centrifuge** at 10,000 g (rcf) for 1 min. Discard the liquid flow through. Repeat until all supernatant has passed through the column. (This will take 3x washes through the column). <br>

-   Note: DNA is selectively bound to the silica membrane in the *MB Spin Column* device in the high salt solution. Contaminants pass through the filter membrane, leaving only DNA bound to the membrane. <br>

-   20:18 BR pipetted 675 ul of the sample CL supernatant to the CL spin filter

-   20:19 BR pipetted 675 ul of the sample 01 supernatant to the 01 spin filter

-   20:20 BR pipetted 675 ul of the sample 02 supernatant to the 02 spin filter

-   20:21 BR pipetted 675 ul of the sample 03 supernatant to the 03 spin filter

-   20:21 BR pipetted 675 ul of the sample 04 supernatant to the 04 spin filter

-   20:22 BR pipetted 675 ul of the sample 05 supernatant to the 05 spin filter

-   20:23 BR pipetted 675 ul of the sample 06 supernatant to the 06 spin filter

-   20:23 BR pipetted 675 ul of the sample 07 supernatant to the 07 spin filter

-   Finished one full box of 1000 ul filter tip pipette tips- JM retrieved a new one

-   20:23 BR pipetted 675 ul of the sample 08 supernatant to the 08 spin filter

-   20:25 BR pipetted 675 ul of the sample 09 supernatant to the 09 spin filter

-   20:26 BR pipetted 675 ul of the sample 10 supernatant to the 10 spin filter

-   20:27 BR pipetted 675 ul of the sample 11 supernatant to the 11 spin filter

-   20:28 JM centrifuged the samples for 1 minute at 10,000 g

-   BR discarded flow through. JM created a new liquid waste container beaker and labeled it "2/28/2024 JM/NM/BR Gut Microbiome Waste Container Do Not Mix with Bleach"

b\. Second Wash: Pipette 675 𝜇l of the supernatant to the same *Spin Filter* and **centrifuge** at 10,000 g (rcf) for 1 min. Discard the liquid flow through. <br>

-   20:37 BR pipetted 675 ul of the sample CL supernatant to the CL spin filter

-   20:38 BR pipetted 675 ul of the sample 01 supernatant to the 01 spin filter

-   20:39 BR pipetted 675 ul of the sample 02 supernatant to the 02 spin filter

-   20:40 BR pipetted 675 ul of the sample 03 supernatant to the 03 spin filter

-   20:40 BR pipetted 675 ul of the sample 04 supernatant to the 04 spin filter

-   20:41 BR pipetted 675 ul of the sample 05 supernatant to the 05 spin filter

-   20:41 BR pipetted 675 ul of the sample 06 supernatant to the 06 spin filter

-   20:42 BR pipetted 675 ul of the sample 07 supernatant to the 07 spin filter

-   20:43 BR pipetted 675 ul of the sample 08 supernatant to the 08 spin filter

-   20:44 BR pipetted 675 ul of the sample 09 supernatant to the 09 spin filter

-   20:45 BR pipetted 675 ul of the sample 10 supernatant to the 10 spin filter

-   20:45 BR pipetted 675 ul of the sample 11 supernatant to the 11 spin filter

-   20:47 JM centrifuged samples for 1 min

-   20:48 BR discarded the flow through of samples from second wash

c\. Third Wash: Pipette 675 𝜇l of the supernatant to the same *Spin Filter* and **centrifuge** at 10,000 g (rcf) for 1 min. Discard the liquid flow through. <br>

-   20:52 BR pipetted the remainder of the sample CL supernatant to the CL spin filter

-   20:53 BR pipetted the remainder of sample 01 supernatant to the 01 spin filter

-   20:54 BR pipetted the remainder of sample 02 supernatant to the 02 spin filter

-   20:54 BR pipetted the remainder of sample 03 supernatant to the 03 spin filter

-   20:55 BR pipetted the remainder of sample 04 supernatant to the 04 spin filter

-   20:56 BR pipetted the remainder of sample 05 supernatant to the 04 spin filter

-   20:57 BR pipetted the remainder of sample 06 supernatant to the 06 spin filter

-   20:57 BR pipetted the remainder of sample 07 supernatant to the 07 spin filter

-   20:58 BR pipetted the remainder of sample 08 supernatant to the 08 spin filter

-   20:58 BR pipetted the remainder sample 09 supernatant to the 09 spin filter

-   20:59 BR pipetted the remainder of sample 10 supernatant to the 10 spin filter

-   20:59 BR pipetted the remainder of sample 11 supernatant to the 11 spin filter

-   21:01 JM centrifuged samples for 1 min

-   21:03 JM discarded the flow through of samples from third wash

<!-- -->

### Protocol Step 16

-   Add 500 𝜇l of *Solution C5* to the filter and **centrifuge** at 10,000 g (rcf) for 30 s. Discard the liquid flow through. <br>

-   Note: *Solution C5* is an ethanol-based wash solution used to further clean the DNA that is bound to the silica filter membrane in the *Spin Filter*. This wash solution removes residual salt, humic acid, and other contaminants while allowing the DNA to stay bound to the silica membrane. <br>

-   21:05 JM added 500 ul of C5 to sample CL

-   21:05 JM added 500 ul of C5 to sample 01

-   21:06 JM added 500 ul of C5 to sample 02

-   21:06 JM added 500 ul of C5 to sample 03

-   21:06 JM added 500 ul of C5 to sample 04

-   21:06 JM added 500 ul of C5 to sample 05

-   21:06 JM added 500 ul of C5 to sample 06

-   21:07 JM added 500 ul of C5 to sample 07

-   21:07 JM added 500 ul of C5 to sample 09

-   21:08 JM added 500 ul of C5 to sample 10

-   21:08 JM added 500 ul of C5 to sample 11

-   21:09 NM centrifuged all the samples at 10,000 g (rcf) for 30 seconds

-   21:11 NM discarded the flow through

### Protocol Step 17

-   **Centrifuge** the *Spin Filter* column a second time at 10,000 g (rcf) for 1 min. <br>

-   Note: This flow-through fraction is non-DNA organic and inorganic waste removed from the silica *Spin Filter* membrane by the ethanol wash solution. The second spin removes residual *Solution C5* (ethanol wash solution). It is critical to remove all traces of wash solution because the ethanol in *Solution C5* can interfere with many downstream DNA applications, such as PCR and gel electrophoresis. <br>

-   21:14 NM re-emptied the flow through just to be sure

-   21:16 NM centrifuged all the samples at 10,000 g (rcf) for 1 minute

### Protocol Step 18

-   Place *Spin Filter* into a clean and labeled *2 ml Collection Tube* (this is the tube going into the freezer, so make sure it's labeled appropriately). Avoid splashing any *Solution C5* onto the *Spin Filter*.<br>
-   This is the final tube so it should be **a low-bind tube**
-   21:28 NM transferred the spin filters to low-bind tubes.
-   21:25 JM performed a test of centrifuging a spin filter with water in a low-bind tube to see if it would leak. It did not leak.

### Protocol Step 19

-   Incubate *Solution C6* for 5 minutes at 65ºC. Then, add 100 𝜇l of *Solution C6* to the center of the filter membrane. <br>

-   Note: Placing *Solution C6* in the center of the small white membrane will make sure the entire membrane is wet. This will result in a more efficient and complete release of the DNA from the Spin Filter membrane. As *Solution C6* passes through the silica membrane, DNA that was bound in the presence of high salt is selectively released by *Solution C6* (which is actually 10 mM Tris), which lacks salt. <br>

-   21:26 Began incubating Solution C6 at 65C on the heat block.

-   21:31 Finished incubating Solution C6

-   21:32 JM added 100 ul of Solution C6 to the filter membrane of sample 08

-   21:33 JM added 100 ul of Solution C6 to the filter membrane of sample 07

-   21:33 JM added 100 ul of Solution C6 to the filter membrane of sample 06

-   21:33 JM added 100 ul of Solution C6 to the filter membrane of sample 05

-   21:33 JM added 100 ul of Solution C6 to the filter membrane of sample 04

-   21:33 JM added 100 ul of Solution C6 to the filter membrane of sample 03

-   21:34 JM added 100 ul of Solution C6 to the filter membrane of sample 02

-   21:34 JM added 100 ul of Solution C6 to the filter membrane of sample 10

-   21:34 JM added 100 ul of Solution C6 to the filter membrane of sample CL

-   21:34 JM added 100 ul of Solution C6 to the filter membrane of sample 09

-   21:34 JM added 100 ul of Solution C6 to the filter membrane of sample 01

-   21:35 JM added 100 ul of Solution C6 to the filter membrane of sample 11

### Protocol Step 20

-   Incubate samples at room temperature for 25 minutes. Then **centrifuge** at 10,000 g (rcf) for 30 sec, and finally discard the spin filter. <br> Your DNA is ready for use! <br>
-   21:36 Began incubating all samples at room temperature.
-   22:05 NM centrifuged the samples in a batch of 9 and a batch of 3 for a minute each. (NM mistakenly set the centrifuge to 1 minute instead of 30 seconds).
-   22:13 JM placed the extracted samples on the bottom shelf of the -20C freezer.
-   22:13 NM discarded the used tubes in the biohazard waste.
-   22:14 JM parafilmed the top of the waste beaker and left it in the BSC.
-   22:20 We wiped down the BSC, extraction station, and centrifuge. We turned off the BSC and centrifuge.

### Protocol Step 21

-   Measure the dsDNA concentration using the **Quantus**, and sample A260/A280 using the **NanoDrop** **\[Quantitation Workstation\]**. Record these details in your lab notebook. <br> Place the properly labeled DNA elutions in the **-20ºC freezer**. Be sure to record where in the freezer you've placed them, and that they are all together in a single, clearly labeled container/rack.

Group members present: JM, RHS, NM

Documented by JM

-   11:05 NM/JM cleaned blench w/ bleach, water, ETOH
-   11:10 NM/RHS cleaned P1000 pipette with bleach, water, ETOH
-   11:11 RHS made a working solution
-   11:20 RHS opened a new 20ul tip box
-   11:20 dye taken out of box
-   11:21 dye added to a working solution
-   11:22 dye w/ working solution vortexed for 5s
-   11:22 dye is placed in the 4C freezer
-   11:23 RHS makes the blank
-   11:25 RHS notices we do not have enough 0.5ml PCR tubes (only 4)- CS autoclaves roughly 50 PCR tubes.
-   11:28 RHS 1x TE buffer adding to 0.5ml PCR tube, add 200ul of the working solution = blank solution (labeled B), vortexed
-   11:29 RHS added 2ul of DNA standard solution, added 200ul of the working solution = standard solution (labeled S), vortexed
-   11:33 RHS prepares sample CL to be quantified (sample, dye working solution, vortex) and placed back on ice
-   11:35 RHS prepares sample 01 to be quantified (sample, dye working solution, vortex) and placed back on ice
-   11:36 RHS prepares sample 02 to be quantified (sample, dye working solution, vortex) and placed back on ice
-   11:37 RHS samples sitting at room temperature for 5 minutes
-   11:42 RHS ends time samples should be sitting at room temperature
-   11:43 RHS selects the dsDNA protocol
-   11:48 RHS calibrates Quantus: Blank, Standard, Clicks "Save"
-   11:49 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample CL.
-   11:49 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 01.
-   11:50 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 02.

**Quantus Final Concentrations (03/04/2024):**

Standard: 102 ng/ul

Blank: 0.0057 ng/ul

Sample CL: 0.0909 ng/ul

Sample 01: 7.1 ng/ul

Sample 02: 2.98 ng/ul

**Quantus Final Concentrations (03/08/2024):**

Group members present: JM, BR, RHS

Documented by JM/BR

-   16:50 RHS/JM cleaned blench w/ bleach, water, ETOH

-   16:50 RHS/JM cleaned P1000 pipette with bleach, water, ETOH

-   16:55 RHS made a working solution

-   16:56 Dye taken out of box

-   16:57 Dye added to a working solution

-   16:57 Dye w/ working solution vortexed for 5s

-   16:57 Dye is placed in the 4C freezer

-   17:00 JM makes the blank

-   17:01 JM 1x TE buffer adding to 0.5ml PCR tube, add 200ul of the working solution = blank solution (labeled B), vortexed.

-   17:03 JM added 2ul of DNA standard solution, added 200ul of the working solution = standard solution (labeled S), vortexed.

-   17:09 RHS prepares sample 03 to be quantified (sample, dye working solution, vortex) and placed back on ice

-   17:12 JM prepares sample 04 to be quantified (sample, dye working solution, vortex) and placed back on ice

-   17:15 BR prepares sample 05 to be quantified (sample, dye working solution, vortex) and placed back on ice

-   17:18 RHS prepares sample 06 to be quantified (sample, dye working solution, vortex) and placed back on ice

-   17:20 RHS prepares sample 07 to be quantified (sample, dye working solution, vortex) and placed back on ice

-   17:21 JM prepares sample 08 to be quantified (sample, dye working solution, vortex) and placed back on ice

-   17:23 JM prepares sample 09 to be quantified (sample, dye working solution, vortex) and placed back on ice

-   17:25 BR prepares sample 10 to be quantified (sample, dye working solution, vortex) and placed back on ice

-   17:29 BR prepares sample 11 to be quantified (sample, dye working solution, vortex) and placed back on ice

-   17:30 JM samples sitting at room temperature for 5 minutes

-   17:35 JM ends time samples should be sitting at room temperature

-   17:35 RHS selects the dsDNA protocol

-   17:36 RHS calibrates Quantus: Blank, Standard, Clicks "Save"

-   17:36 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 03.

-   17:37 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 04.

-   17:38 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 05.

-   17:39 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 06.

-   17:39 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 07.

-   17:40 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 08.

-   17:41 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 09.

-   17:41 RHS vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 10.

-   17:42 JM vortex, spins down/ wipe the outside of the tube with a kimwipe, measures fluorescence of the unknown sample. JM records the final sample concentration of sample 11.

-   17:44 JM/BR wiped down the bench w/ bleach, water, ETOH and turned the Quantus off.

Standard: 100 ng/ul

Blank: 0.0056 ng/ul

Sample 03: 5.2 ng/ul

Sample 04: 15 ng/ul

Sample 05: 14 ng/ul

Sample 06: 21 ng/ul

Sample 07: 15 ng/ul

Sample 08: 8.3 ng/ul

Sample 09: 6.1 ng/ul

Sample 10: 19 ng/ul

Sample 11: 3.97 ng/ul

**Nanodrop Final Concentrations (03/04/2024):**

Group members present: JM, NM

Documented by JM

-   12:17 NM loads Blank (DNAse-free water)

-   12:19 NM Sample CL: 1.8 ng/ul, 260:0.36, A260/280: 1.61, A260/A230: 0.13

-   12:20 NM Sample 01: 14.3 ng/ul, 260: 0.286, A260/280: 1.71, A260/A230: 1.03

-   12:21 NM Sample 02: 6.5 ng/ul, 260: 0.130, A260/280: 1.71, A260/A230: 0.66

-   12:22 NM Sample 03: 5.8 ng/ul, 260: 0.115, A260/280: 1.65, A260/A230: 0.64

-   12:23 NM Sample 04: 11.3 ng/ul, 260: 0.226, A260/280: 1.86, A260/A230: 1.03

-   12:24 NM Sample 05: 11.6 ng/ul, 260: 0.232, A260/280: 1.78, A260/A230: 0.91

-   12:25 NM Sample 06: 29.5 ng/ul, 260: 0.590, A260/280: 1.79, A260/A230: 1.33

-   12:26 NM Sample 07: 14.5 ng/ul, 260: 0.291, A260/280: 1.72, A260/A230: 1.02

-   12:27 NM Sample 08: 12.5 ng/ul, 260: 0.249, A260/280: 1.78, A260/A230: 0.96

-   12:28 NM Sample 09: 7.8 ng/ul, 260: 0.156, A260/280: 1.78, A260/A230: 0.70

-   12:29 NM Sample 10: 18.9 ng/ul, 260: 0.377, A260/280: 1.77, A260/A230: 1.37

-   12:30 NM Sample 11: 6.8 ng/ul, 260: 0.137, A260/280: 1.68, A260/A230: 0.70

-   12:32 JM/NM wiped down the bench with bleach, water, ETOH and turned the Nanodrop off.
